src/controllers/hello_controller.js

```diff
@@ -0,0 +1,34 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+    static targets = [ "firstName", "lastName", "sexMale", "sexFemale", "age", "firstNameDisplay", "lastNameDisplay", "ageDisplay", "genderDisplay", "formContainer", "displayContainer" ]
+
+   
+    getAndDisplayPatientDetails() {
+        // validation of Patient details form
+        if(this.firstNameTarget.value == ""){return alert("Please enter first name")}
```

Comment:
Create a separate validation function. Validation code should not be part of this function.